### PR TITLE
[MOD-2248] Add two optional arguments `valueToSum` and `parameters` for logEvent

### PR DESCRIPTION
**Pull request by danghy**
@hansemannn  I updated PR #26 with the latest code. I also opened a Jira ticket here: [https://jira.appcelerator.org/browse/AC-3533](https://jira.appcelerator.org/browse/AC-3533)

### CODE REVIEW

File: android/src/facebook/TiFacebookModule.java [hunk 1]
```diff
@@ -259,11 +259,17 @@ public void onCompleted(GraphResponse response) {
...
+	public void logCustomEvent(String event, @Kroll.argument(optional = true) Double valueToSum, @Kroll.argument(optional = true) KrollDict parameters) {
 		Activity activity = TiApplication.getInstance().getCurrentActivity();
 		AppEventsLogger logger = AppEventsLogger.newLogger(activity);
+		Bundle paramBundle = parameters != null ? Utils.mapToBundle(parameters) : null;
+		Log.d(TAG, "logCustomEvent: "+ event + " valueToSum: " + valueToSum);
 		if (logger != null) {
-			logger.logEvent(event);
+			if (valueToSum == null){
+				logger.logEvent(event, paramBundle);
+			}else{
```
Comment (hansemannn) [inline android/src/facebook/TiFacebookModule.java]: Missing spaces in `if (){` and `}else{`. Also please remove the debug log for now. Thanks!

**Comment by hansemannn** [comment]
~~I also would change the iOS implementation a bit (will provide demo code here) and~~ iOS looks fine, we can leave it like it is! You just need to update the docs to accept the optional parameters.

**Comment by danghy** [comment]
@hansemannn Just updated the code and docs. Please check and let me know if you need to change anything :)

**Comment by hansemannn** [comment]
@danghy Please resolve the merge conflicts before.

**Comment by danghy** [comment]
@hansemannn  done :)

**Comment by hansemannn** [comment]
Approved!

**Pull request merged**